src/toxcore/binary_io.rs

```diff
@@ -101,10 +101,9 @@ macro_rules! encode_decode_test (
     ($test:ident, $value:expr) => (
         #[test]
         fn $test() {
+            #[allow(unused_imports)]
```

Comment:
Why not remove this `use`?

Reply:
Because we use gen_nonce and other crypto functions inside of this macro